tests/quantization/test_quantized_module.py

```diff
@@ -469,27 +469,44 @@ def quantized_module_predictions_are_equal(
 @pytest.mark.parametrize(
     "model_class, input_shape",
     [
-        pytest.param(FC, (100, 32 * 32 * 3)),
-        pytest.param(partial(CNN, input_output=3), (100, 3, 32, 32)),
-        pytest.param(partial(CNNMaxPool, input_output=3), (100, 3, 32, 32)),
+        pytest.param(FC, (100, 32 * 32 * 3), id="FC"),
+        pytest.param(partial(CNN, input_output=3), (100, 3, 32, 32), id="CNN"),
+        pytest.param(partial(CNNMaxPool, input_output=3), (100, 3, 32, 32), id="CNNMaxPool"),
+        pytest.param(
+            partial(EmbeddingModel, num_embeddings=100, embedding_dim=3),
+            (100, 3),
+            id="EmbeddingModel",
+        ),
     ],
 )
-def test_serialization(model_class, input_shape):
+@pytest.mark.parametrize(
+    "rounding_threshold_bits", [None, 8, {"n_bits": 8, "method": "APPROXIMATE"}]
+)
+def test_serialization(model_class, input_shape, rounding_threshold_bits):
     """Test the serialization of quantized modules."""
 
     # Define the torch model
     torch_fc_model = model_class(activation_function=nn.ReLU)
     torch_fc_model.eval()
 
     # Create a random input
-    numpy_input = numpy.random.uniform(size=input_shape)
-    torch_input = torch.from_numpy(numpy_input).float()
+    if isinstance(model_class, partial) and model_class.func == EmbeddingModel:
```

Comment:
I don't think you need to add this test here , imo you test anything new ! This tests is about serializing QMs, with a few torch models (in order to get a relevant initialized QM). I don't see what `EmbeddingModel` brings more to the test, you already (rightly) test its compilation in `test_compile_torch` anyway ! imo it's safe to remove that part

if you were thinking about the QM rounding serialization issue, imo the `rounding_threshold_bits` fix you've added below is enough

Reply:
EmbeddingModel is the only model that will create a onnx preprocessing which we need to make sure can be serialised.

Reply:
hmm, sure ok, I first didn't find this relevant because it's the same kind of object than onnx_model, but you're right we never now it that changes later so better to be safe, sorry for the noise !